README.zh-cn.md

```diff
@@ -0,0 +1,120 @@
+### [简介](https://github.com/sz3/cimbar) | [关于](https://github.com/sz3/cimbar/blob/master/ABOUT.md) | [CFC](https://github.com/sz3/cfc) | LIBCIMBAR
+### [详细内容](DETAILS.md) | [演示](PERFORMANCE.md) | [下一步](TODO.md)
+
+## libcimbar: 彩色图形矩阵条形码
+
+一种用于气隙数据传输的实验性条形码格式。
+
+只需一个电脑显示器和一个智能手机摄像头，它就可以维持850Kbit/s（约106 KB/s）的速度！
+
+<p align="center">
+<img src="https://github.com/sz3/cimbar-samples/blob/v0.6/b/4cecc30f.png" width="70%" title="A non-animated mode-B cimbar code" >
+</p>
+
+## 为什么？
+
+编码器将动画条形码输出到计算机或智能手机屏幕：
+* 编码器web应用程序：https://cimbar.org
+
+虽然解码器是一个使用手机摄像头读取动画条形码的手机应用程序：
+* 解码器安卓应用程序：https://github.com/sz3/cfc
+
+未使用互联网/蓝牙/NFC/等，所有数据都通过相机镜头传输。你可以自己试试，或者相信我的话，它真的有效。:)
+
+## 工作原理？
+
+`cimbar`是一种高密度的二维条形码格式。数据存储在彩色图块网格中——根据选择哪个图块和选择哪种颜色来绘制图块，对比特进行编码。对数据应用Reed-Solomon纠错，以解释视频->数字解码的有损性。预计错误率低于1%，并已纠正。
+
+`libcimbar`，这个优化版本，包括一个基于喷泉代码（wirheir）和zstd压缩的简单文件编码协议。高达33MB（压缩后）的文件以一系列cimbar代码编码，可以输出为图像或实时视频馈送。一旦成功解码了足够多的不同图像帧，文件将被成功重建和解压缩。即使图像接收顺序错误，或者有些图像已损坏或丢失，这也是正确的。
```

Comment:
Is "wirheir" a typo?

Reply:
Yes, my carelessness. I'll check the translation again.